lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/ActionForm.js

```diff
@@ -0,0 +1,312 @@
+/* ************************************************************************
+   Copyright: 2021 OETIKER+PARTNER AG
+   License:   GPLv3 or later
+   Authors:   Tobi Oetiker <tobi@oetiker.ch>
+              Fritz Zaucker <fritz.zaucker@oetiker.ch>
+   Utf8Check: äöü
+************************************************************************ */
+
+/**
+ * Action Form Widget.
+ */
+qx.Class.define("callbackery.ui.plugin.ActionForm", {
+    extend : qx.ui.container.Composite,
+    /**
+     * create a page for the View Tab with the given title
+     *
+     * @param cfg {Object} plugin configuration map
+     * @param getFormData {Function} method to get form data
+     */
+    construct : function(cfg,getFormData) {
```

Comment:
what is the use case for 'getFormData' ?

Reply:
Ah, dumm benannt. Es ist parentFormData ... umbenannt.